gobblin-service/src/main/java/org/apache/gobblin/service/modules/template/StaticFlowTemplate.java

```diff
@@ -176,10 +178,10 @@ public boolean isResolvable(Config userConfig, DatasetDescriptor inputDescriptor
       try {
         this.jobSpecResolver.resolveJobSpec(jobSpecBuilder.withTemplate(template).build());
       } catch (JobTemplate.TemplateException | ConfigException | SpecNotFoundException exc) {
-        return false;
+        return exc;
```

Comment:
See comment earlier. We probably can throw the Exception instead of catching here, and let the caller handle the exception.